metadrive/policy/manual_control_policy.py

```diff
@@ -25,9 +25,12 @@ def __init__(self, obj, seed):
         if config["manual_control"]:
             if config["controller"] == "keyboard":
                 self.controller = KeyboardController(pygame_control=pygame_control)
-            elif config["controller"] == "joystick":
+            elif config["controller"] in ["joystick", "xboxController"]:
                 try:
-                    self.controller = SteeringWheelController()
+                    if config["controller"] == "joystick":
+                        self.controller = SteeringWheelController()
+                    elif config["controller"] == "xboxController":
```

Comment:
"xboxController" might cause error when typing, try use ["xboxController", "xboxcontroller", "xbox"] and more options that the user might want to use

Reply:
updated